client/src/test/kotlin/com/lowes/auditor/client/infrastructure/frameworks/mapper/JsonNodeMapperTest.kt

```diff
@@ -0,0 +1,272 @@
+package com.lowes.auditor.client.infrastructure.frameworks.mapper
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.lowes.auditor.core.entities.domain.Element
+import com.lowes.auditor.core.entities.domain.ElementMetadata
+import com.lowes.auditor.core.entities.domain.EventType
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.nulls.shouldNotBeNull
+import io.kotest.matchers.shouldBe
+import io.kotest.matchers.shouldNotBe
+import io.kotest.matchers.string.shouldStartWith
+
+/**
+ * Unit Tests for [JsonNodeMapper]
+ */
+class JsonNodeMapperTest : BehaviorSpec({
```

Comment:
@bettyand  - Thanks for the contribution.

One comment around tests validation, rather than validating each and every field specifically, how about we write a result json for the expected structure and all the expected fields just like how you are creating simpleJson and complex, nested json.. and then use the expectedJson to match agains the actualJson. This way its easy to visualize the result for future engineers who work on this or also for reviewers to see if things are working as expected. Please follow the same approach for all tests in this file.

After we fix it, its LGTM from my side.